<!-- -->
<launch>
  <arg name="sensor_frame" default="/lidar_base" />
  <arg name="input0" default="/front/velodyne_points" />
  <arg name="input1" default="/rear/velodyne_points" />
  <arg name="output" default="/points_concat" />

  <node pkg="points_preprocessor" type="points_concat_filter"
        name="points_concat_filter" output="log">
    <param name="sensor_frame" value="$(arg sensor_frame)" />
    <remap from="/lidar0/points_raw" to="$(arg input0)" />
    <remap from="/lidar1/points_raw" to="$(arg input1)" />
    <remap from="/points_concat" to="$(arg output)" />
  </node>
</launch>
